docs/src/usage/scenario_generation.md

```diff
@@ -115,7 +115,29 @@ ADRIA.fix_factor!(dom;
     coral_cover_tol=1.0
 )
 ```
+The ranges for sampling can also be constrained if running scenarios for a particular 
+range of a parameter is of interest. For example, if one wanted to investigate only 
+scenarios with high fogging and seeding, the following example could be used:
 
+```julia
+dom = ADRIA.load_domain()
+
+# Adjust seeding bounds. Note only lower and upper bounds are needed because the factors in  
+# question have a uniform distribution.
+ADRIA.set_factor_bounds!(dom, :N_seed_TA, (500000.0, 1000000.0 + 1.0))
+ADRIA.set_factor_bounds!(dom, :N_seed_CA, (500000.0, 1000000.0 + 1.0))
+ADRIA.set_factor_bounds!(dom, :N_seed_SA, (500000.0, 1000000.0 + 1.0))
+
+# Adjust fogging bounds. Note lower, upper and mode parameters are needed because it 
+# is a triangular distribution.
+ADRIA.set_factor_bounds!(dom, :fogging, (0.2, 0.3, 0.1))
+
+# Adjust multiple factors in one function evaluation
```

Comment:
I know it is one "function evaluation" but it's not the usual language in this context